### PR TITLE
clarify fixing common test issues

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
@JensTimmerman last commit needed for corretc rpm building

File: lib/vsc/install/shared_setup.py [hunk 1]
```diff
@@ -144,13 +144,13 @@ def _log(self, level, msg, args):
...
-VERSION = '0.9.8'
+VERSION = '0.9.9'
 
 log.info('This is (based on) vsc.install.shared_setup %s' % VERSION)
 
-# list of non-vsc packages that need python- prefix for correct rpm dependencies
+# list of non-vsc packages that do not need python- prefix for correct rpm dependencies
 # vsc packages should be handled with clusterbuildrpm
-PREFIX_PYTHON_BDIST_RPM = ('setuptools',)
+NO_PREFIX_PYTHON_BDIST_RPM = ()
```
Comment (JensTimmerman) [inline lib/vsc/install/shared_setup.py]: this looks confusing, you might want to change `()` to `tuple()` so it's clear what this is supposed to be?

Comment (JensTimmerman) [inline lib/vsc/install/shared_setup.py]: actually, can't this be a list? So you can maybe extend it later on, or is this exactly what you don't want?
in that case, maybe document that?

**Comment by JensTimmerman** [comment]
jenkins: test this please

**Pull request merged**